test/functional/plugin/man_spec.lua

```diff
@@ -131,5 +131,32 @@ describe(':Man', function()
                                                           |
       ]])
     end)
+
+    it('handles : characters in input', function()
+      -- what input looks like on terminal
+      -- [30m    0  [31m    1  [32m    2  [33m    3  
+      -- [34m    4  [35m    5  [36m    6  [37m    7  [90m    8  [91m    9  
+      -- [92m   10  [93m   11  [94m   12  [95m   13  [96m   14  [97m   15  
+      -- [38:5:16m   16  
+
+      -- [40m    0  [41m    1  [42m    2  [43m    3  
+      -- [44m    4  [45m    5  [46m    6  [47m    7  [100m    8  [101m    9  
+      -- [102m   10  [103m   11  [104m   12  [105m   13  [106m   14  [107m   15  
+      -- [48:5:16m   16  
```

Comment:
This is a good type of comment, but in this case it's redundant with the `rawfeed()` call so I removed it.